spec/acceptance/lesson_email_spec.rb

```diff
@@ -10,13 +10,17 @@
     @teacher = FactoryGirl.create(:user)
     @attendee = FactoryGirl.create(:user)
     @venue = FactoryGirl.create(:venue)
-    @lesson = FactoryGirl.create(:next_month_lesson, start_time: Date.current + 1.month + 1.day + 19.hours, start_time: Date.current + 1.month + 1.day + 21.hours)
+    @lesson = create(
```

Comment:
Yes, [this line makes it possible](https://github.com/rails-school/school/blob/7d57949f803be1190b78f21bc55fa3089abdd17e/spec/support/factory_girl.rb#L2), and the tests are cleaner and require less typing this way.